src/components/NavBar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Home from './Home';
+import About from './About';
+import Contact from './Contact';
+
+export default function NavBar() {
+  return (
+    <Routes>
+      <Route path="*/home*" element={<Home />} />
```

Comment:
**Optional** 
You can replace the path= ''/home " by  '/' so that it is considered as the  home page on each render